### PR TITLE
ZER-141-Change_button

### PR DESCRIPTION
**Pull request by malk1ss**
After prod 
after_forPRODuk
after dev
after_forDEVuk
https://jira.softserve.academy/browse/ZER-141

Description

Hide "Register", "Login", "Contact us"  buttons if env is production.

Add new section with link to "Contacts" from main site

### CODE REVIEW

File: app/helpers/contacts_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module ContactsHelper
+end
```
Comment (opazyniuk) [inline app/helpers/contacts_helper.rb:2]: remove

File: app/views/layouts/_navigation.html.slim [hunk 2]
```diff
@@ -1,18 +1,23 @@
...
+    = link_to t('.contacts'), "https://zerowastelviv.org.ua/kontakty/", class: 'tab'
+  .tabs
+  - if Rails.env.production?
+    = link_to t('.contacts'), "https://zerowastelviv.org.ua/kontakty/", class: 'tab'
+  .donate-btn
+    = link_to t('.donate'), "https://zerowastelviv.org.ua/pidtrymaty/", target: '_blank'
+  - if current_locale?(:en)
+    = link_to "Українська", locale_path(:id => :uk), method: :patch, class: "btn btn-outline-success btn-sm"
+  - else
+    = link_to "English", locale_path(:id => :en), method: :patch, class: "btn btn-outline-success btn-sm"
```
Comment (opazyniuk) [inline app/views/layouts/_navigation.html.slim:23]: endline

File: app/controllers/contacts_controller.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class ContactsController < ApplicationController
+  def index
+  end
+end
```
Comment (opazyniuk) [inline app/controllers/contacts_controller.rb:4]: remove

**Review by opazyniuk — CHANGES_REQUESTED** [review]

**Review by PrincTwilig — APPROVED** [review]

**Comment by denysiuk** [comment]
Please resolve the conflicts in config/locales/en.yml and config/locales/uk.yml and fill in the info about the task: summary of issue and summary of change

Screenshot 2022-07-31 at 13 26 29

File: app/javascript/css/main.css.scss [hunk 4]
```diff
@@ -34,8 +34,8 @@ a {
   flex-wrap: wrap;
   align-items: center;
   border-bottom: 2px solid $gray;
-  padding-left: 11.88%;
-  padding-right: 11.88%;
+  padding-left: 13.5%;
+  padding-right: 13.5%;
```
Comment (PrincTwilig) [inline app/javascript/css/main.css.scss:38]: can you change padding-left and padding-right to 9%

Comment (opazyniuk) [reply app/javascript/css/main.css.scss:38]: remove this changes

File: app/javascript/css/calculator.css.scss [hunk 5]
```diff
@@ -98,8 +98,12 @@
...
 }
 
+.logo-zerowaste {
+  width: 140px;
+}
+
 .logo-zerowaste img {
   aspect-ratio: 16 / 9;
-  width: 80%;
+  width: 100%;
```
Comment (opazyniuk) [inline app/javascript/css/calculator.css.scss:107]: ???

**Review by opazyniuk — CHANGES_REQUESTED** [review]

**Review by opazyniuk — CHANGES_REQUESTED** [review]
please add screenshots with before/after your changes

**Review by DmitriyShevtsov — APPROVED** [review]

**Review by Oleksii-Farat — APPROVED** [review]

**Review by opazyniuk — CHANGES_REQUESTED** [review]
conflicts

File: spec/features/sign_out_spec.rb [hunk 6]
```diff
@@ -2,6 +2,7 @@
 
 require 'rails_helper'
 
+if RAILSENVDEV
```
Comment (opazyniuk) [inline spec/features/sign_out_spec.rb:5]: this is wrong decition

File: spec/features/visit_calculator_spec.rb [hunk 7]
```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
-
+if RAILSENVDEV
```
Comment (opazyniuk) [inline spec/features/visit_calculator_spec.rb:4]: this is wrong decition

File: spec/rails_helper.rb [hunk 8]
```diff
@@ -80,3 +80,5 @@
     with.library :rails
   end
 end
+
+RAILSENVDEV = Rails.env.development?
```
Comment (opazyniuk) [inline spec/rails_helper.rb:84]: this is wrong decition

File: spec/features/admin/navigator_spec.rb [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+        before do
+          visit root_path
+        end
+        it 'when redirect to link' do
+          expect(page).to have_link('contacts')
+          click_link('contacts', href: 'https://zerowastelviv.org.ua/en/contacts/')
+        end
+      end
+    end
+  end
```
Comment (opazyniuk) [inline spec/features/admin/navigator_spec.rb:29]: endline

**Review by opazyniuk — CHANGES_REQUESTED** [review]

File: app/views/layouts/_navigation.html.slim [hunk 10]
```diff
@@ -2,14 +2,25 @@ body
...
+            = link_to t('.contacts'), "https://zerowastelviv.org.ua/kontakty/", class: 'tab', id: 'contacts'
       - else
-        = link_to t('.sign_up'), new_user_registration_path, class: 'tab'
-        = link_to t('.log_in'), new_user_session_path, class: 'tab', id: 'log_in'
-      = link_to t('.contact_us'), new_message_path, class: 'tab'
+        .tabs
+            - if current_locale?(:en)
+              = link_to t('.contacts'), "https://zerowastelviv.org.ua/en/contacts/", class: 'tab', id: 'contacts'
+            - else
+              = link_to t('.contacts'), "https://zerowastelviv.org.ua/kontakty/", class: 'tab', id: 'contacts'
```
Comment (opazyniuk) [inline app/views/layouts/_navigation.html.slim:23]: add link as localization to avoid usage of this if

**Review by opazyniuk — CHANGES_REQUESTED** [review]

File: spec/features/admin/navigator_spec.rb [hunk 11]
```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+  RSpec.feature 'Сontacts css', type: :feature do
+    describe 'contacts css on entry page', js: true do
+      context 'when entry page' do
+        before do
+          visit root_path
+        end
+        it 'css' do
```
Comment (opazyniuk) [inline spec/features/admin/navigator_spec.rb:9]: it's not ok to describe test in this way

File: spec/features/admin/navigator_spec.rb [hunk 12]
```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+  RSpec.feature 'Сontacts css', type: :feature do
```
Comment (opazyniuk) [inline spec/features/admin/navigator_spec.rb:3]: what this "Contacts css" means? tests should have meaningful description

**Review by opazyniuk — CHANGES_REQUESTED** [review]

**Review by opazyniuk — APPROVED** [review]

**Pull request merged**